packages/sfextract.vm/sfextract.vm.nuspec

```
@@ -4,11 +4,12 @@
     <id>sfextract.vm</id>
     <version>2.1.0.20250219</version>
     <authors>Joery Droppers</authors>
-    <description>command-line utility to extract files from single file bundles in .NET</description>
+    <description>Sfextract is a command-line utility to extract files from single file bundles in .NET.</description>
```

The tool name is `sfextract` (command line) or `SingleFileExtractor` (name used in GH repo), but not `Sfextract`.

```suggestion
    <description>sfextract extracts contents (assemblies, configuration, etc.) from .NET single file bundles.</description>
```